app/models/file_set.rb

```diff
@@ -1,5 +1,4 @@
-# Generated by curation_concerns:models:install
+# Generated by hyrax:models:install
 class FileSet < ActiveFedora::Base
   include ::Hyrax::FileSetBehavior
-  self.indexer = FileSetIndexer
 end
```

Comment:
Is this indexer a custom addition? Should it be removed?  If it's removed here, does it get included somewhere else in the stack?